src/azure-cli/azure/cli/command_modules/appservice/_help.py

```diff
@@ -1246,9 +1246,15 @@
   - name: turn on "alwaysOn"
     text: >
         az webapp config set -g MyResourceGroup -n MyUniqueApp --always-on true
-  - name: turn on "alwaysOn" through a json with content '{"alwaysOn", true}'
+  - name: for non-PowerShell users, turn on "alwaysOn" through a json with content '{"alwaysOn", true}'
     text: >
         az webapp config set -g MyResourceGroup -n MyUniqueApp --generic-configurations '{"alwaysOn": true}'
+  - name: for PowerShell users, turn on "alwaysOn" through a json with content '{"alwaysOn", true}' (note the backslashes escaping the double quotes)
```

Comment:
@seligj95 can you review the text & provide the blob you'd like to use here in the help. Thanks!